pom.xml

```diff
@@ -502,7 +503,8 @@
                 <requireJavaVersion>
                   <version>[1.7,)</version>
                 </requireJavaVersion>
-                <dependencyConvergence/>
+                <dependencyConvergence>
+                </dependencyConvergence>
```

Comment:
Why are you breaking this apart into start/end tags?

Reply:
No particular reason. It's a random change that happened as I was experimenting with different options to try to make this work.